vbox-2012r2.json

```diff
@@ -4,7 +4,10 @@
     "vboxmanage": [
       [ "modifyvm", "{{.Name}}", "--natpf1", "guest_winrm,tcp,,55985,,5985" ],
       [ "modifyvm", "{{.Name}}", "--memory", "2048" ],
-      [ "modifyvm", "{{.Name}}", "--cpus", "2" ]
+      [ "modifyvm", "{{.Name}}", "--cpus", "2" ],
+      [ "modifyvm", "{{.Name}}", "--vram", "36" ],
+      [ "modifyvm", "{{.Name}}", "--chipset", "ich9" ],
+      [ "storageattach", "{{.Name}}", "--storagectl", "IDE Controller", "--port", "1", "--device", "0", "--type", "dvddrive", "--medium", "additions" ]

```

Comment:
You could also use "guest_additions_mode": attach to do this automatically via Packer. I may open a pull request with 'attach' since I'm running into a build failure with upload.